cluster/aws/util.sh

```diff
@@ -352,6 +352,34 @@ function wait-for-instance-running {
   done
 }
 
+function allocate-elastic-ip {
+  $AWS_CMD allocate-address --domain vpc --output text | cut -f3
```

Comment:
@madis 

Recommendation (from the style guide you referenced, yes that's the correct one) is to always use `${...}` for variables with more than one character to make that consistent regardless of whether you have some string following the variable that would require the curly braces, so we essentially try to use it everywhere...

If you expect `${AWS_CMD}` to expand to a single word then you should double quote it, but your example of `ls -la` shows that maybe you don't...

In that case, if `${AWS_CMD}` is defined inside your script, you should probably use a bash array instead since that's the more general way to support a variable expanding to multiple arguments that is friendly with whitespace.

Though, if you expect AWS_CMD to be set or exported from the external environment, then it's fair to just use the unquoted `${AWS_CMD}` as you're using it. In that case, though, make sure you handle the way of `${AWS_CMD}` not being set in the environment before this script is called, for instance by defaulting it to something useful, or by checking that it is properly defined before you use it (or at least defaulting it to something like `false` which under `set -o errexit` will trigger a somewhat more useful error when executed instead of taking the next argument and assuming it's a command, in which case you run a risk if that actually looks like a valid command then something unintended will run instead of what you originally expected.)

I hope this addresses your concerns.